facefusion/jobs/job_manager.py

```diff
@@ -199,32 +200,26 @@ def set_steps_status(job_id : str, step_status : JobStepStatus) -> bool:
 
 def read_job_file(job_id : str) -> Optional[Job]:
 	job_path = find_job_path(job_id)
-
-	if is_file(job_path):
-		with open(job_path, 'r') as job_file:
-			return json.load(job_file)
+	if is_json(job_path):
+		return read_json(job_path) #type:ignore[return-value]
```

Comment:
read_json should return None anyway and have the exception handling, so no need to use is_json() here after some refactoring